README.md

```diff
@@ -72,7 +72,92 @@ You can generate it using `BundleAnalyzerPlugin` with `generateStatsFile` option
 command: `webpack --profile --json > stats.json`
 
 `webpack-bundle-analyzer --help` will show you all usage information.
- 
+
+## Contributing
+
+To contribute to `webpack-bundle-analyzer`, fork the repository and clone it to your machine. [See this GitHub help page for what forking and cloning means](https://help.github.com/articles/fork-a-repo/)
+
+### Setup packages
+
+Then install [`yarn`](https://yarnpkg.com/):
+
+```sh
+npm install --global yarn
+```
+
+Next, install this package's dependencies with `yarn`:
+
+```sh
+yarn install --pure-lockfile
```

Comment:
Oh, seems like it was fixed in some 0.17.x versions of `yarn` as I got some changes before. Or maybe it was because you had updated `yarn.lock` recently. But that's just great! Less confusing instructions, hooray!